src/main/java/com/netflix/imflibrary/st2067_2/IMFCompositionPlaylistType.java

```diff
@@ -66,7 +66,24 @@
     private final List<IMFEssenceDescriptorBaseType> essenceDescriptorList;
     private final IMFErrorLogger imfErrorLogger;
     private final String coreConstraintsVersion;
-    private final String applicationId;
+    private final Set<String> applicationIdSet;
+
+
+    @Deprecated
```

Comment:
Add Javadoc explaining why the old method was deprecated, and what new signature should be used instead.
https://docs.oracle.com/javase/tutorial/java/annotations/predefined.html